graphql/database/missions/MissionModel.js

```diff
@@ -99,6 +100,16 @@ class Mission extends BaseModel {
       endOfMissionAwards: [],
     }
   }
+
+  static get permissions() {
```

Comment:
this permissions block isn't needed with the override

Reply:
Cool, sounds good.